dataloader/players.py

```diff
@@ -46,4 +46,21 @@ def load(args):
 		print("filtering scorer points >= {}".format(scorer_points))
 		json_object = list(filter(lambda x: scorer_points < x['assists'] + x['goals_scored'], json_object))
 
+	# sort_by
+	if args.get('sort_by') is not None and args.get('sort_by') != '' and len(json_object) > 0 and args.get('sort_by') in json_object[0]:
```

Comment:
I would follow the Google Python styleguide ([Link](https://google.github.io/styleguide/pyguide.html)) and split lines which are longer than 80 characters into multiple lines, e.g.

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/65946090/95293683-34ccd600-0874-11eb-8713-abe5adbd45da.png">
or in this case:

```python
if (args.get('sort_by') is not None and
    args.get('sort_by') != '' and
    len(json_object) > 0 and
    args.get('sort_by') in json_object[0]):
```

Also, I'm pretty sure you can shorten the statement above as shown below. It makes the code less readable though..

```python
if (args.get('sort_by') and # this checks for is not None and != ''
    len(json_object) > 0 and
    args.get('sort_by') in json_object[0]):
```